### ISSUE TITLE
Gérer le back

### ISSUE THREAD

**Issue by JC144** [JC144/EDF_Simulateur_Prix#48]
Actuellement, la navigation va vers l'avant mais on ne peut pas revenir sur les écrans précédents sans rafraichir le site

**Comment by juliencarnot** [comment]
oui, c'est assez perturbant de ne pas pouvoir revenir en arrière pour affiner/modifier les données entrée (par ex. baisse de puissance souscrite ou changement d'horaires HC) sans réactualiser/recharger la page.

**Comment by JC144** [comment]
Fixé dans la prochaine mise à jour.
